gravitee-apim-console-webui/src/services-ngx/audit.service.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { HttpClient } from '@angular/common/http';
+import { Inject, Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Audit } from '../entities/audit/audit';
+
+import { Constants } from '../entities/Constants';
+import { MetadataPage } from '../entities/MetadataPage';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuditService {
+  constructor(private readonly http: HttpClient, @Inject('Constants') private readonly constants: Constants) {}
+
+  listByOrganization(): Observable<MetadataPage<Audit>> {
+    return this.http.get<MetadataPage<Audit>>(`${this.constants.org.baseURL}/audit`);
```

Comment:
I think it should be plural here `/audits` and so here too https://github.com/gravitee-io/gravitee-api-management/pull/1659/files#diff-ae8e4ed2b9141528c8dbec5a2e3e70893e64f60804b68c387c77d75c38540527R224 no?

Reply:
Ok, it's consistant with the other url (ApiAudit) but it's weird 🤔

Reply:
not strange to have plural only for org audits ?

Reply:
Yes 🤷‍♂️